src/commands/generate.ts

```diff
@@ -142,6 +142,7 @@ export default class extends Command {
 
         const style = ctx.client.getHordeStyle(style_raw)
 
```

Comment:
Same thing as the previous comment

Reply:
Assuming you mean clip skip at line 145 (not shown for some reason), 1 is the default as mentioned in the comment above, but I'll set it to undefined here as well.

Reply:
Same thing as above, remove the ?? undefined